pyflow/host.py

```diff
@@ -283,7 +281,7 @@ def preamble_init(self, ecflowpath):
 
         return script
 
-    def preamble_error_function(self, ecflowpath, additional_commands=[]):
```

Comment:
What was the `additional_commands` used for? I've never come across it, is that from a previous iteration of this logic?

Reply:
I don't know but it was not interfaced nor used in the other functions... I guess the goal was to add something similar to what we're doing here but it's hard to have a really generic way of doing this because the functionality could depend on where it's defined in the preamble (for instance, exit_hook is used in two places).